src/Anchoring.js

```diff
@@ -92,15 +92,21 @@ function Anchoring (params) {
     const block = await _(this).provider.getBlock(height)
     if (block === null) return status.block(0)
 
-    const blockValid = verifyBlock(block, validatorKeys)
-    if (!blockValid) return status.block(1, { block })
+    try {
+      await verifyBlock(block, validatorKeys)
```

Comment:
To avoid such as constructions I used `await to` syntax
look:
https://github.com/exonum/exonum-client-anchoring/blob/b4aec95d6b1216c8c98e90b5cc13c78b3dc19407/src/Anchoring.js#L34
https://github.com/exonum/exonum-client-anchoring/blob/b4aec95d6b1216c8c98e90b5cc13c78b3dc19407/src/common/common.js#L6

Reply:
fixed, thanks